MAINTAINERS

```diff
@@ -21,7 +21,7 @@ David Schneiderbauer <dschneiderbauer@gmail.com> (@daviian)
 Peter Žeby <morlinest@gmail.com> (@morlinest)
 Matti Ranta <matti@mdranta.net> (@techknowlogick)
 Michael Lustfield <mtecknology@debian.org> (@MTecknology)
-Jonas Franz <info@jonasfranz.software> (@JonasFranzDEV)
```

Comment:
There are some other changes to this file such as MTecknology is no longer a maintainer, would you be able to copy/paste all the changes from main repo as you are making these updates?